python-threatexchange/threatexchange/exchanges/clients/utils/common.py

```diff
@@ -0,0 +1,17 @@
+from requests.adapters import HTTPAdapter
```

Comment:
nit: I git an internal nagger that forces me to add 

`# Copyright (c) Meta Platforms, Inc. and affiliates.` to the tops of all files, though I can do it as a followup